### ISSUE TITLE
I encounterd an "AssertionError"

### ISSUE THREAD

**Issue by GoogleCodeExporter** [waterxi/svnplot#20]
```
What steps will reproduce the problem?
1. i had installed all the required softwares.
2. then when i type "python svnlog2sqlite.py http://192.168.1.105/test
/usr/local/svn.db" 
3.

What is the expected output? What do you see instead?
ERROR:root:Error
Error
Traceback (most recent call last):
  File "svnlog2sqlite.py", line 37, in convert
    rootUrl = self.svnclient.getRootUrl()
  File "/usr/lib/python2.5/site-packages/svnplot/svnlogiter.py", line 352,
in getRootUrl
    self.getRootUrl2()
  File "/usr/lib/python2.5/site-packages/svnplot/svnlogiter.py", line 326,
in getRootUrl2
    assert(len(changepathlist) > 0)
AssertionError


What version of the product are you using? On what operating system?
svnplot-0.5.11  on debian5.0 python2.5 svn1.5.1

Please provide any additional information below.


```

Original issue reported on code.google.com by `xiaofeng...@gmail.com` on 22 Nov 2009 at 2:55

**Comment by GoogleCodeExporter** [comment]
```
thanks for the bug report. I need some more details to reproduce it. 

If you type the URL in the browser(http://192.168.1.105/test), what do you see 
? Do you get list of folders from the SVN ?



```

Original comment by `nitinbh...@gmail.com` on 22 Nov 2009 at 3:07

**Comment by GoogleCodeExporter** [comment]
```
yes,i can get a list.before i test svnplot,i already took a test.if you need 
more
information,i"ll provide as soon as you tell me.
```

Original comment by `xiaofeng...@gmail.com` on 23 Nov 2009 at 1:10

**Comment by GoogleCodeExporter** [comment]
```
Thanks for the update. Please enable the 'logging' in svnlog2sqlite (use -g 
option) and run it on the repository. Please attach the log file to this issue. 

The log file will help me understand the problem in more details.

thanks.
```

Original comment by `nitinbh...@gmail.com` on 23 Nov 2009 at 3:29

**Comment by GoogleCodeExporter** [comment]
```
i got the logfile,but i don't think this is useful.
```

Original comment by `xiaofeng...@gmail.com` on 24 Nov 2009 at 2:03

Attachments:
- [svnlog2sqlite.log](https://storage.googleapis.com/google-code-attachments/svnplot/issue-20/comment-4/svnlog2sqlite.log)

**Comment by GoogleCodeExporter** [comment]
```
looks like 'logging' also has a bug. :-(. I will check both.
```

Original comment by `nitinbh...@gmail.com` on 24 Nov 2009 at 4:31

**Comment by GoogleCodeExporter** [comment]
```
I still could not reproduce this error. Is there any publicly available 
repository where you can reproduce the error. Or can you create a dummy 
repository with 5/6 revisions and send me the repository as zip.


```

Original comment by `nitinbh...@gmail.com` on 30 Nov 2009 at 4:57

**Comment by GoogleCodeExporter** [comment]
```
Many issues related to loc and unknown node kind and assertion errors were 
fixed 0.7.5. Hence for the time being marking it as fixed 
```

Original comment by `nitinbh...@gmail.com` on 28 May 2011 at 1:52
- Changed state: **Fixed**
